cmp/97c9a2ee-5775-42e9-b39d-f82209a0ccce/component.lp

```diff
@@ -0,0 +1,69 @@
+(librepcb_component 97c9a2ee-5775-42e9-b39d-f82209a0ccce
+ (name "USB Connector 3.0 Micro B")
+ (description "Connector with 11 pins (SuperSpeed Powered-B)\n\n- USB 3.0 Micro B")
+ (keywords "usb,usb3,superspeed,micro-b,powered-b")
+ (author "Danilo Bargen")
+ (version "0.1")
+ (created 2019-10-15T20:36:20Z)
+ (deprecated false)
+ (category c4c12c6e-abcd-4856-8445-e96c2737423e)
+ (schematic_only false)
+ (default_value "{{ PARTNUMBER or DEVICE or COMPONENT }}")
+ (prefix "J")
+ (signal 270d71f0-783d-4318-9278-1d9473ffb168 (name "GND") (role passive)
+  (required true) (negated false) (clock false) (forced_net "")
+ )
+ (signal 2571f771-934a-4c88-858c-19ab1bd1e512 (name "VBUS") (role passive)
+  (required true) (negated false) (clock false) (forced_net "")
+ )
+ (signal 6f3146e3-2870-4901-bc5a-9b2f6e32457b (name "GND_DRAIN") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 48c3f388-69cc-4a9c-b969-4bb2e4bc6531 (name "SSTX+") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal ec365cdf-e6da-436e-b5ae-b1ac90827885 (name "DPWR") (role passive)
+  (required true) (negated false) (clock false) (forced_net "")
```

Comment:
Is DPWR really required to be connected? If not:

```suggestion
  (required false) (negated false) (clock false) (forced_net "")
```

Reply:
It would be really strange to use a "powered B" connector and then not connect the power lines... What do you think, should I remove the requirement?

Reply:
I'm not an USB expert. But if it's totally valid to leave these pins unconnected, then yes, I'd remove the requirement. Actually for most kinds of connectors there are almost no pins required since there are a lot of different use-cases (compared to passive components or ICs).

Reply:
It depends what "totally valid" means. I don't think it conforms to the spec to put a USB C connector on a PCB and use it only for powering your board with GND/VBUS (without having connected any data pins and without doing any power negotiation), however someone may still use a USB C connector for exactly this purpose.

In the case of Powered B there's probably no reason to use a Powered Micro USB connector versus a Non-Powered Micro USB connector if those additional pins are not being used, but on the other hand we should probably not control/decide how people should be able to use USB connectors. If they want to have an USB connector for mechanical stability only (and no pins connected), then why not.

I'll remove all pin connection requirements.

Reply:
> In the case of Powered B there's probably no reason to use a Powered Micro USB connector versus a Non-Powered Micro USB connector if those additional pins are not being used

Sometimes in industry one uses a "wrong" part because of non-technical reasons (lower price, higher availability, already used in other designs, requirement from customer, ...).

> I'll remove all pin connection requirements.

:+1: